test/unit/test_normalizer.py

```diff
@@ -53,6 +46,53 @@ def test_normalizer(rules_def, normalized):
     assert expected == results_sorted
 
 
+@pytest.mark.parametrize(
+    "rules_def, normalized",
+    [
+        (
+            [
+                {
+                    "name": "not_null",
+                    "column": "a",
+                    "separate_time_filters": [{"column": "c"}],
+                }
+            ],
+            [{"name": "not_null", "column": "a", "time_filter": {"column": "c"}}],
+        ),
+        (
+            [
+                {
+                    "name": "not_null",
+                    "column": "a",
+                    "separate_time_filters": [
+                        {"column": "a", "days": 10},
+                        {"column": "b", "days": 5},
+                    ],
+                }
+            ],
+            [
+                {
+                    "name": "not_null",
+                    "column": "a",
+                    "time_filter": {"column": "a", "days": 10},
```

Comment:
Here the result is not `[]`, but e.g. in this test https://github.com/kiwicom/contessa/pull/31/files#diff-e089fed83b649e67fb48ed22e1a539beR31 - there is list. Could the type mismatch make a problem somewhere ? I would rather unify it if possible.

Reply:
Good catch, fixing.